services/orders.js

```diff
@@ -1,8 +1,96 @@
 const moment = require('moment');
 const { Op } = require('sequelize');
 const { Order, OrderItem } = require('../models');
+const { take, countBy, meanBy, chain, sortBy } = require('lodash');
 
-function storeOrder({ grantAmount, orderReason, postcodeArea, items }) {
+function getAllOrders() {
+    return Order.findAll({
+        order: [['updatedAt', 'DESC']],
+        include: [
+            {
+                model: OrderItem,
+                as: 'items'
+            }
+        ]
+    }).then(orders => {
+        const normalisedDateFormat = 'YYYY-MM-DD';
+        // the date we added dropdown grant size options rather than free text
+        const LAUNCH_DATE = '2018-02-01';
```

Comment:
Probably not – after this date, values are either picked from a list or entered as "other" (meaning they could look like the older entries which were always manually typed). Bit hacky but in a few months time (eg. 1st July or so) this will stop being a problem as all records before this date will have been deleted.